setup.py

```diff
@@ -115,7 +115,8 @@ def run_tests(self):
         'pyyaml',
         'argparse',
         'argcomplete',
-        'future'
+        'future',
+        'pyBarcode==0.8b1'
```

Comment:
I fired some issues on viivakoodi but then after saw that the steenzout fork was way more maintained. I think this is the best option, I will try to change dependencies to this one.